src/Functions/DateTimeTransforms.h

```diff
@@ -23,12 +23,16 @@ namespace DB
 static constexpr auto microsecond_multiplier = 1000000;
 static constexpr auto millisecond_multiplier = 1000;
 
+using DT_OVERFLOW_MODE = FormatSettings::DateTimeOverflowMode;
```

Comment:
That imports `DT_OVERFLOW_MODE` into the `DB` namespace. Not very pretty. I think prefixing the enum in the few places where it is used is okay.